src/models/media-container/search-result/directory/library-section-title.yaml

```
@@ -0,0 +1,5 @@
+type: object
+properties:
+  librarySectionTitle:
+    type: string
+    example: "Movies" 
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)